native/.idea/runConfigurations.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why these changes?

Reply:
No idea, IntelliJ was quite persistent in re adding those even though I deleted them quite a few times :D